aion/lib/aion/ranking.ex

```diff
@@ -11,26 +11,41 @@ defmodule Aion.Ranking do
     UserCategoryScore
   }
 
-  @type category_score_t :: %{category_id: integer, category_name: String.t, scores: (list user_score_t)}
```

Comment:
Not sure if it's a good idea to format the files that you don't touch. 

I'm 100% for running `mix format` on the ones that you edit anyway. As far as I remember the syntax is just `mix format filepath/folderpath`. Or you can `mix format` and then checkout the files that you didn't touch.

Reply:
Yeah, did not notice while I was playing with formatter, will revert that.